bundler/lib/bundler/checksum.rb

```diff
@@ -0,0 +1,228 @@
+# frozen_string_literal: true
+
+module Bundler
+  class Checksum
+    DEFAULT_ALGORITHM = "sha256"
+    DEFAULT_BLOCK_SIZE = 16_384
+    private_constant :DEFAULT_BLOCK_SIZE
```

Comment:
should `DEFAULT_ALGORITHM` be private? and also maybe an array?

Reply:
I'll make it private which will protect our ability to change it later if we need an array.